csharp/Svix/Models/SvixClientOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Svix.Models
+{
+    public sealed class SvixClientOptions
+    {
+        public string AccessToken { get; set; }
+        
+        public string ServiceUrl { get; set; }
```

Comment:
```suggestion
        public string ServerUrl { get; set; }
```

Reply:
We name this field ServerUrl in our other libraries so we should be consistent here